packages/@tinacms/graphql/src/resolver/index.ts

```diff
@@ -706,9 +712,21 @@ export class Resolver {
       case 'boolean':
       case 'number':
       case 'reference':
-      case 'image':
         accumulator[field.name] = value
         break
+      case 'image':
+        if (this.config) {
+          if (this.config.useRelativeMedia === true) {
+            accumulator[field.name] = value
+          } else {
+            accumulator[
+              field.name
+            ] = `https://assets.tinajs.io/${this.config.clientId}/${value}`
```

Comment:
@kldavis4 Is this `assets.tina.io` now?  Sounded like that was an easy change to make.

Reply:
it _should_ be - just waiting for the prod deploy to complete